lib/gen_fsm.ex

```diff
@@ -20,69 +85,82 @@ defmodule GenFSM do
     end
   end
 
+  @spec stop(server, reason :: term, timeout) :: :ok
   def stop(fsm, reason \\ :normal, timeout \\ :infinity) do
     :gen.stop(fsm, reason, timeout)
   end
 
+  @type fsm_ref :: pid | name
```

Comment:
and then @type fsm_ref should be the same as @server above, and we can just eliminate @server?